production/packaging/build_and_test_all_in_docker

```diff
@@ -48,10 +49,12 @@ declare -i SKIP_PRECOMMIT=0
 declare -i NO_TESTS=0
 declare -i NO_PLATFORM_BUILD=0
 declare -a AMI_REGIONS
-declare AWS_IMAGE_TAG
```

Comment:
Why is this going away?

Reply:
Good point, that's my mistake.
I copy and pasted the our dev version of `build_and_test_all_in_docker` then reverted non-azure changes (i.e. diff upstream 3.10 vs upstream 3.4) so that we get azure changes efficiently. I guess I missed this non-azure changes to revert. I'll fix

Reply:
Fixed in 56c3ea3. Cherry-picked it for takurosato/rebase-dev-v3.10.0 as well